pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/FixedByteChunkSVForwardIndexReader.java

```diff
@@ -48,6 +52,175 @@ public ChunkReaderContext createContext() {
     }
   }
 
+  @Override
+  public void readValuesSV(int[] docIds, int length, int[] values, ChunkReaderContext context) {
+    if (getValueType().getStoredType().isFixedWidth() && !_isCompressed && isContiguousRange(docIds, length)) {
+      switch (getValueType().getStoredType()) {
```

Comment:
(minor)
`getStoredType()` is redundant because value type is stored type for forward index. Same for other places